hi-ml-multimodal/src/health_multimodal/image/utils.py

```diff
@@ -0,0 +1,53 @@
+#  -------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  -------------------------------------------------------------------------------------------
+
+import tempfile
+from pathlib import Path
+
+from torchvision.datasets.utils import download_url
+
+from .. import REPO_URL
+from .model import ImageModel
+from .inference_engine import ImageInferenceEngine
+from .data.transforms import create_chest_xray_transform_for_inference
+
+IMAGE_WEIGHTS_NAME = ""  # TODO
```

Comment:
Why isn't this in `health_multimodal/__init__.py` along with the text model?

Reply:
I was having some circular imports issues and wasn't able to put it where I wanted: in `image/__init__.py`. I can try moving it up the tree, but I don't think it's too important. Feel free to commit a fix!

Reply:
I've moved them in f7cd899. Thanks!